ovos_plugin_manager/templates/audio.py

```diff
@@ -3,26 +3,31 @@
 These classes can be used to create an Audioservice plugin extending
 OpenVoiceOS's media playback options.
 """
-from ovos_bus_client.message import Message
+from abc import ABCMeta, abstractmethod
 
-from ovos_plugin_manager.templates.media import AudioPlayerBackend as _AB
 from ovos_utils import classproperty
 from ovos_utils.log import log_deprecation
-from ovos_utils.ocp import PlaybackType, TrackState
+from ovos_utils.fakebus import FakeBus
 from ovos_utils.process_utils import RuntimeRequirements
 
 log_deprecation("ovos_plugin_manager.templates.audio has been deprecated on ovos-audio, "
                 "move to ovos_plugin_manager.templates.media", "0.1.0")
 
 
-class AudioBackend(_AB):
+class AudioBackend(metaclass=ABCMeta):
```

Comment:
nit, we dont really need to subclass from metaclass=ABCMeta

Reply:
reverted with try/except import